coresdk/src/coresdk/basics.cpp

```diff
@@ -94,4 +95,19 @@ namespace splashkit_lib
     {
         return std::stod( text );
     }
+    
+    bool is_char(const string &text)
+    {   
+        string s = trim(text);
+        
+        return s.length()==1;
+    }
+    
+    bool is_bool(const string &text)
+    {
+        string s = to_lowercase(trim(text));
+        
+        return s == "true" || s == "t" || s == "false" || s = "f" || s == "0" || s == "1"; 
```

Comment:
```suggestion
        return s == "true" || s == "t" || s == "false" || s == "f" || s == "0" || s == "1"; 
```
This time this line didn't compile as it was missing an extra equals sign